Vonk.Plugin.ExampleOperation/VonkPluginService.cs

```diff
@@ -6,7 +6,10 @@
 using Hl7.Fhir.Model;
 using Vonk.Fhir.R3;
 using Task = System.Threading.Tasks.Task;
+using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Vonk.Plugin.ExampleOperation.Tests")]
```

Comment:
This is usually done in a ./Properties/AssemblyInfo.cs class, see e.g. Vonk.Core project.